### ISSUE TITLE
I just need some help.

### ISSUE THREAD

**Issue by ghost** [allistoncarlos/homebridge-servo-switch#3]
Hello, I want to use this to build a smart door lock. The Servo turnes the key in the lock kind of thing. The Servo is connected to an ESP8266. I used the ino file on the raspberry and your config.json part on my Homebridge. The problem now is, that whenever i try to controll it via Homebridge, the device says it is offline and Homebridge log says, that no Power_url is defined. If you could help me, it would be really appreciated. Maybe contact me here or via my eMail Adress: jakob.matz@tenesmos.de.
Thankyou very much :)

**Comment by allistoncarlos** [comment]
Hi @JakobYooo! Thanks for using the plugin! I'm glad that it could help others achieve different goals.
And sorry for the delay in replying... Here are some questions to help you find the reason of the problem:

- Are you using the .ino file on the ESP8266? 
- The Homebridge is installed on the Raspberry Pi? 
- Did you set the powerOn and powerOff URLs in the config.json? 
- What happens when you try to access the URLs via browser in your local network?

Let me know if one of these questions help you correct the problem.

Best Regards!

**Comment by ghost** [comment]
Yes, all the point are done. I tried accessing the URL from a browser, but the connection times out.
Have you got Skype or Discord, if you would be fine with this?
